### ISSUE TITLE
Import files in other folders/paths

### ISSUE THREAD

**Issue by luiz-manesco** [hilkoc/vbaDeveloper#59]
Is there a way to import a file that is located in other path which is not the "src" folder when using **testImport()**? It would be very useful to edit VBA files remotely

**Comment by zanniboni** [comment]
i don't think this is possible considering the actual version of the code but it would be great
